show/main.py

```diff
@@ -1057,5 +1057,19 @@ def ecn():
     proc = subprocess.Popen(cmd, stdout=subprocess.PIPE, shell=True)
     click.echo(proc.stdout.read())
 
+#
+# 'reboot-cause' command ("show reboot-cause")
+#
+@cli.command('reboot-cause')
+def reboot_cause():
+    """Show cause of most recent reboot"""
+    PREVIOUS_REBOOT_CAUSE_FILE = "/var/cache/sonic/previous-reboot-cause.txt"
```

Comment:
>previous- [](start = 51, length = 9)

Who prefix "previous-" to the file name?

Reply:
The mechanism uses two files: "reboot-cause.txt" which is used to writing the cause of the *next* reboot, and "previous-reboot-cause.txt" which is used for reading the cause of the previous reboot. "reboot-cause.txt" initially gets created with the the text "Unexpected reboot" which will become the reboot reason in the event of power loss. This is done in rc.local.

Reply:
By reading your reply, I understand the design fully. Could you move them to code comment so other developer could easily maintain based on your thought?

---
In reply to: [197602125](https://github.com/Azure/sonic-utilities/pull/271#discussion_r197602125) [](ancestors = 197602125)